pages/glossary/non-repeated-content.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Non-repeated content
+key: non-repeated-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+A node is _non-repeated content_ if all the following are true:
+
+- the node is [perceivable content][]; and
+- the node is not part of any [block of repeated content][]; and
+- the node is after (in [tree order][] in the [flat tree][]) at least one [block of repeated content][].
```

Comment:
This implies that a document that has no block of repeated content (and therefore no repeated content) has no "non-repeated content" (and therefore has repeated content which is a contradiction!) :-)

```suggestion
- the node is after (in [tree order][] in the [flat tree][]) of at least one [block of repeated content][].
```

Reply:
Yes, as mentioned in https://github.com/act-rules/act-rules.github.io/pull/421#issuecomment-743120298 the name "non-repeated content" is fairly bad but I couldn't find better…

If a page has no "block of repeated content", then it also has no "non-repeated content" either (which is why this is a good name).
A correct name would be "non-repeated perceivable content located after some repeated content" but that is too long to be really usable…

I'm open for suggestions for a better name for this definition, I agree this one is confusing.

For the rules (and the SC), we do want "non-repeated content located after some repeated content" since non-repeated content located at the start of the page doesn't require any means of jumping to it. So I'm confident that the definition is was the rule needs, but that the name is bad…

Reply:
I agree that the definition is correct. I wouldn't mind "non-repeated content after repeated content", but I'm not going to block approving the rule based on this :-)